packages/hummingbird/src/templates/components/CampaignConfirmed/FundraiserText.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import qs from 'qs';
+import { FormattedMessage } from 'react-intl';
+import Title from './Title';
+import Text from './Text';
+
+export default ({ location }) => {
+  const status = qs.parse(location.search.replace('?', '')).transaction_status;
+  switch (status) {
+    case 'complete':
+      return (
+        <React.Fragment>
+          <FormattedMessage
+            id="hummingbird.CampaignConfirmed.fundraiserMain.successTitle"
```

Comment:
`hummingbird.CampaignConfirmed.main.fundraiserSuccessTitle` would make more sense here, as the main component is consistent across campaign types.

Same goes for the other IDs in here.